fastlane/jetpack_metadata/ja/subtitle.txt

```diff
@@ -1 +1 @@
-WordPress サイトをもっと速く、もっと安全に。
\ No newline at end of file
+より速く安全な WordPress
```

Comment:
17 characters ✅ 

That is if App Store Connect does something like `"...".length`, which would make sense to me given that the 30 characters limitation is not a matter of sentence complexity but of screen real estate. I'm not actually sure if Japanese strings kanji are counted in the same way as we do with the letters in a Roman language.